src/Messaging/src/RabbitMQ/Support/DefaultMessageHeadersConverter.cs

```diff
@@ -95,11 +95,6 @@ public virtual void FromMessageHeaders(IMessageHeaders headers, RC.IBasicPropert
                 target.ContentType = source.ContentType();
             }
 
-            if (string.IsNullOrEmpty(target.ContentType))
-            {
-                target.ContentType = RabbitHeaderAccessor.DEFAULT_CONTENT_TYPE;
```

Comment:
@dtillman setting default this in messaging code, causes issues in the Streams code with its default content type detection. I removed this here and in other places, All the tests are passing so I assume there is no issue, but do you see any problem with this?

Reply:
OK .. sounds good ... can't remember why I did that... might want to double check the Java code to make sure it's all right!

Reply:
Yes the Java code was not doing this, was just making sure the messaging aspects did not need this.